flake.nix

```diff
@@ -57,6 +57,7 @@
           ];
 
         CARGO_TARGET_DIR = "target_dirs/nix_rustc";
+        NVIM_RUST_ENABLED_FEATURE_FLAGS = "full-ci";
```

Comment:
Do we actually want to push this globally? I would expect to have it set per-user and just inherit it during the nix initialization, but maybe I'm misunderstanding an aspect of the way the `flake.nix` file is used.

Reply:
Summarizing from meeting:
- example config for neovim in `.nvim`
- source `.nvim/config.sh` in a shellhook for nix shell
- readme for user about this

Reply:
Conceptually this can't work from the `flake.nix`, because nix shells only know about staged or committed files.

I'll need to get nvim to search from the root directory of the project for a `.nvim/env.sh`.

Reply:
🙀

Reply:
Should be good to go 👍